src/library_egl.js

```diff
@@ -232,7 +232,7 @@ var LibraryEGL = {
       {{{ makeSetValue('value', '0', 'EGL.contextAttributes.antialias ? 1 : 0' /* Multisampling enabled */, 'i32') }}};
       return 1;
     case 0x3033: // EGL_SURFACE_TYPE
-      {{{ makeSetValue('value', '0', '0x0004' /* EGL_WINDOW_BIT */, 'i32') }}};
+      {{{ makeSetValue('value', '0', '0x4' /* EGL_WINDOW_BIT */, 'i32') }}};
```

Comment:
these look good, but i'm pretty sure they get the unneeded 0s removed anyhow in the js optimizer?

Reply:
Yeah, that's true. Though there were one or two locations where the hex value appears in a printed string, so thought to be consistent.